package/MDAnalysis/analysis/MSD.py

```diff
@@ -0,0 +1,452 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+MSD --- :mod:'MDAnalysis.analysis.MSD'
+==============================================================
+
+:Author: Andrew Biedermann
+:Year: 2016
+:Copyright: GNU Public License v3
+
+This tool uses all available data to calculate the mean squared displacement
+(MSD) as a function of time, averaging over all atoms in the atom selection
+and all restarts. The MSD of any atom selection can be calculated,
+as any atoms which remain within the atom group over an entire time interval
+are included in the calculation, allowing for highly specific, position-
+dependent MSD calculations. The tool is robust to zero-length atom selections,
+and allows for the calculation of multiple atom selections simultaneously.
+Additional command line functionality is built in to allow for easy use
+in a batch/slurm scheduler and parallelization of especially large jobs.
+
+Important Note: Calculations are run on the trajectory as given to the tool,
+so some preprocessing may be required to properly account for periodic
+boundary conditions. For example, if calculating the MSD of a Gromacs
+simulation, the following preprocessing is necessary:
+    >>> gmx trjconv -f mysim.xtc -s mysim.tpr -o processed_sim.xtc -pbc nojump
+
+MSD command line usage
+----------------------
+
+This calculates the MSD of 3 atom selections over the first 50000 frames
+of the trajectory:
+
+    >>> ./MSD.py -f mysim.xtc -s mysim.tpr --sel 'name NA,name NA and prop z
```

Comment:
Please remove all mentions and docs of the CLI version. We don't support scripts in MDAnalysis right now.